docs/fsharp/language-reference/fsharp-collection-types.md

```diff
@@ -5,7 +5,7 @@ ms.date: 05/16/2016
 ---
 # F# Collection Types
 
-By reviewing this topic, you can determine which F# collection type best suits a particular need. These collection types differ from the collection types in the .NET Framework, such as those in the `System.Collections.Generic` namespace, in that the F# collection types are designed from a functional programming perspective rather than an object-oriented perspective. More specifically, only the array collection has mutable elements. Therefore, when you modify a collection, you create an instance of the modified collection instead of altering the original collection.
+By reviewing this topic, you can determine which F# collection type best suits a particular need. These collection types differ from the collection types in .NET, such as those in the `System.Collections.Generic` namespace, in that the F# collection types are designed from a functional programming perspective rather than an object-oriented perspective. More specifically, only the array collection has mutable elements. Therefore, when you modify a collection, you create an instance of the modified collection instead of altering the original collection.
```

Comment:
I made a small change unrelated to the issue (the .NET Framework -> .NET), because `System.Collections.Generic` isn't .NET Framework-specific.